attributes/erlang_solutions.rb

```diff
@@ -1,5 +1,5 @@
-default['yum']['erlang_solutions']['baseurl'] = 'http://packages.erlang-solutions.com/rpm/centos/$releasever/$basearch'
-default['yum']['erlang_solutions']['description'] = 'Centos $releasever - $basearch - Erlang Solutions'
+default['yum']['erlang_solutions']['baseurl'] = "http://packages.erlang-solutions.com/rpm/centos/#{platform_version.split('.').first}/$basearch"
```

Comment:
In essence you're asking should it be "7" or "7.2" though, both should work, however, they will exhibit different behaviour in "point release upgrade" situations -

* Using `.to_i` means you'd pull from /7.2 and point upgrades are on the user, this cookbook will only upgrade to /7.3 after the system is also upgraded to that point release.
* Using `.split` means you'd pull from /7 and Chef is likely to **immediately** pick up any packages which are newer and distributed with 7.3 when that is released.

I'm happy to do either but I think if we go with `platform_version.to_i` that you should audit other community cookbooks, including Yum, and converge the approach. I think differing behaviours in different cookbooks would potentially be a poor user experience. /cc @iennae